feedback/feedback.py

```diff
@@ -11,89 +11,104 @@
 import six
 
 from xblock.core import XBlock
-from xblock.fields import Scope, Integer, String, List, Float
+from xblock.fields import Boolean, Dict, Integer, List, Float, Scope, String
+from xblockutils.resources import ResourceLoader
 from web_fragments.fragment import Fragment
 
 
-# We provide default text which is designed to elicit student thought. We'd
+# Make '_' a no-op so we can scrape strings
+_ = lambda text: text
+loader = ResourceLoader(__name__)
+
+
+# We provide default text which is designed to elicit student thought. We"d
 # like instructors to customize this to something highly structured (not
 # "What did you think?" and "How did you like it?".
-DEFAULT_FREEFORM = "What did you learn from this? What was missing?"
-DEFAULT_LIKERT = "How would you rate this as a learning experience?"
-DEFAULT_DEFAULT = "Think about the material, and try to synthesize key " \
-                  "lessons learned, as well as key gaps in our presentation."
-DEFAULT_PLACEHOLDER = "Take a little bit of time to reflect here. " \
-                      "Research shows that a meaningful synthesis will help " \
-                      "you better understand and remember material from this " \
-                      "course."
-DEFAULT_ICON = "face"
+DEFAULT_FREEFORM = _("What did you learn from this? What was missing?")
+DEFAULT_LIKERT = _("How would you rate this as a learning experience?")
+DEFAULT_DEFAULT = _("Think about the material, and try to synthesize key "
+                    "lessons learned, as well as key gaps in our presentation.")
+DEFAULT_PLACEHOLDER = _("Take a little bit of time to reflect here. "
+                        "Research shows that a meaningful synthesis will help "
+                        "you better understand and remember material from this "
+                        "course.")
+DEFAULT_ICON = "num"
 DEFAULT_SCALETEXT = ["Excellent", "Good", "Average", "Fair", "Poor"]
 
 # Unicode alt faces are cute, but we do nulls instead for a11y.
-ICON_SETS = {'face': [""]*5,  # u"😁😊😐😞😭",
-             'num': u"12345",
-             'midface': [""]*5}  # u"😞😐😊😐😞"}
+ICON_SETS = {"face": [""]*5,  # u"😁😊😐😞😭",
+             "num": u"12345",
```

Comment:
From the requirements I understood that we need the 10 choices, aren't we?

Reply:
I didn't refactor code at all, and for now this variable is useless, as range of assessments is used `ASSESSMENTS_NUMBER` variable